scss/main.scss

```diff
@@ -921,21 +911,16 @@ a:hover {
     margin-left: 10rem !important;
   }
 
+
   @media only screen and (max-width: 1225px) {
     width: 10rem !important;
   }
 
-  /*@media only screen and (max-width: 950px) {
-    width: 15rem !important;
-
-    &.polaroid1 {
-      margin-left: 15rem !important;
-    }
-
+  @media only screen and (max-width: 950px) {
```

Comment:
can you center the polaroids on this tablet/narrow desktop view (needed when width is 960px - 1180px)
![Screen Shot 2021-08-19 at 9 51 29 AM](https://user-images.githubusercontent.com/62481939/130080647-a2306ca9-03df-44ec-afaa-9b79ebd54c31.png)